src/month.js

```diff
@@ -90,6 +105,24 @@ export default function CalendarMonth(props) {
               {
                 days.map((day) => (
                   <div key={day.format('YYYYMMDD')} className="tt-cal-day">
+                    { (
```

Comment:
side nit: personally, I'd split this condition out into a `const` flag within the `render()` function. conditionals within jsx more than a few lines long tend to degrade readability, IMO.